File: accessspecifier.cpp
#include<iostream>
using namespace std;
class AccessSpecifierDemo{
    private:
        int priVar;
    protected:
        int proVar;
    public:
        int pubVar;
    void setVar(int priValue ,int proValue ,int pubValue)
    {
        priVar=priValue;
        proVar=proValue;
        pubVar=pubValue;

    }
    void getVar(){
        cout<<"private : " <<priVar<<endl;
        cout<<"protected : " <<proVar<<endl;
        cout<<"public : " <<pubVar<<endl;

    }
};
int main(){
    AccessSpecifierDemo obj;
    obj.setVar(10,30,40);
    obj.getVar();
    return 0;

}

File: accessspecifier.java
class AccessSpecifier{
    private int priVar;
    protected int proVar;
    public int pubVar;
    public void setVar(int priValue ,int proValue ,int pubValue)
    {
        priVar=priValue;
        proVar=proValue;
        pubVar=pubValue;

    }
    public void getVar(){
        System.out.println("private : " +priVar);
        System.out.println("protected : " +proVar);
        System.out.println("public : " +pubVar);


    }
}

class AccessSpecifierDemo{ 
    public static void main(String[] args){
        AccessSpecifier obj=new AccessSpecifier();
        obj.setVar(18,7,45);
        obj.getVar();

    }

}
